pkg/ebpf/tracee.go

```diff
@@ -652,11 +652,19 @@ func (t *Tracee) populateBPFMaps() error {
 	}
 
 	cZero := uint32(0)
-	configVal := make([]byte, 144)
+	configVal := make([]byte, 208)
 	binary.LittleEndian.PutUint32(configVal[0:4], uint32(os.Getpid()))
 	binary.LittleEndian.PutUint32(configVal[4:8], t.getOptionsConfig())
 	binary.LittleEndian.PutUint32(configVal[8:12], t.getFiltersConfig())
 	binary.LittleEndian.PutUint32(configVal[12:16], uint32(t.containers.GetCgroupV1HID()))
+	binary.LittleEndian.PutUint64(configVal[16:24], t.config.Filter.UIDFilter.Less)
```

Comment:
Maybe not for this PR but when I integreate these changes in #1750, but we should probably encapsulate this in the `Set` method.
Probably pass the buffers for the max and min values to the BPF Filter struct. 

As preparation, while I do like the cleaner `Set` maybe we should pass the config or buffer pointers to the Set method for now so it won't get lost, WDYT?

Reply:
The Set method will possibly be removed in a future PR I'm working on for multiple scopes. The reason for that is the need to set values that are a combination of multiple filters. For this reason, ai think we should wait with such changes for now

Reply:
Interesting. So we would have a function similar to `Set` that would take multiple Filters as parameters? Either way, we will have it until then, but I'll defer handling this to my PR.